membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -40,18 +42,18 @@ class AccountController extends LazyLogging {
 
     // TODO - refactor to use the Zuora-only based lookup, like in AttributeController.pickAttributes - https://trello.com/c/RlESb8jG
```

Comment:
As an aside, we should be able to clean up this code significantly once we've done this re-factor. Given that these updates are happening at the account level, we should be able to stop caring about the subscription/plan types altogether, and have a single endpoint which will update a users card for any account with an active sub.